src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) center no-repeat;
+
+    & + & {
+      margin-left: 4px;
+    }
+
+    &--color {
+      background: url(../images/star-active.svg) center no-repeat;
```

Comment:
As above, use the shorthand property for background to improve code readability and maintainability.